cmd/vsphere/main.go

```diff
@@ -34,9 +41,11 @@ func main() {
 	}
 
 	cfg := config.GetConfigOrDie()
-
+	syncPeriod := 10 * time.Minute
```

Comment:
At some point this fix was lost.

Reply:
can you please link the specific commit where this was lost and put it back in its own commit?

Reply:
@enxebre fixed

Reply:
Thanks a log for splitting the commits! fwiw I didn't meant to necessarily cherry-pick back the syncPeriod commit but rather add a link in the description pointing to the commit which dropped it by accident.

Can you please link the counter part PRs for actuator in the PR description and elaborate a bit on the motivation behind this change and also at minimum explain which others providers will be affected by this, e.g openstack/rhv/metal3.

That along with the commits as they are broken down now would have dramatically reduce the friction and time to review this PR in the first place. It would also make extremely easier for people getting here with less context to understand the reasoning behind the change. People with less context includes ourselves in a month from now or doing context switching from other repos.

Reply:
They won't be affected immediately, even if it gets merged, I'll open issues in other repos.

Reply:
mm wouldn't they break as soon as this get merged, as the health check will fail for them when the mao runs?

Reply:
I see, let me open a couple of issues.